services/dashboard/views/model/metrics.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0
+ * See accompanying LICENSE file.
+ */
+'use strict';
+
+angular.module('dashboard.metrics', [])
+
+  .factory('Metrics', [function () {
+    return {
```

Comment:
Add reference here to the data type Meter. Something like:
//TODO datatype: Meter. Try and do serde using scalajs See #458